### DISCUSSION TITLE
Support for Ubuntu 22.04 or RHEL9

### DISCUSSION THREAD

**Discussion by phoenyx762** [StackStorm/st2#6163] [Q&A]
Love Stackstorm... a little concerned Stackstorm is not available for Ubuntu 22.04 or RHEL9. 
Any plans for Debian 12?

**Comment by nzlosh** [comment]
Nice to hear people are loving StackStorm. :+1: 

There is work underway to release v3.9.0 to support newer versions of Ubuntu and RHEL.  You can see the progress here https://github.com/orgs/StackStorm/projects/31/views/1?pane=issue&itemId=43896992.

Keep in mind the StackStorm project is 100% community driven.  If you would like to see Debian 12 support included, you're welcome to create a PR for it, however Python 3.11 support is not a hard requirement for 3.9.0 as Ubuntu Jammy has both 3.10 and 3.11 available so 3.10 support is a hard requirement and 3.11 is a nice to have.

There was an attempt to add Debian Buster (Python 3.7) but it failed because of a threading bug in eventlets which stalled the effort.  https://github.com/StackStorm/st2-packages/pull/628 and https://github.com/StackStorm/st2packaging-dockerfiles/pull/75

There is no fixed ETA for a release date as volunteer time is given on an ad-hoc basis.  A best guess is in 3-6 months time.

If you'd like to join in on the efforts to get 3.9.0 released, please feel free to join the slack community, look at open issues and the likes.
